services/todo-legacy/controllers/todo-controller.ts

```diff
@@ -0,0 +1,43 @@
+import { api, Application, inject } from '@loopback/core';
+import { def } from './todo-controller.api';
+import { TodoRepository } from '../repositories/todo';
+import * as util from 'util';
+
+@api(def)
+export class TodoController {
+  @inject('repositories.todo')
+  repository: TodoRepository;
+
+  constructor() {
+  }
+
+  async getTodo(title?: any) {
+    let filter = title ? { where: { title: title }} : {};
+    return await this.repository.find(null, filter);
+  }
+
+  async getTodoById(id: number) {
+    return await this.repository.find(id);
+  }
+
+  async createTodo(todo) {
```

Comment:
Ah, so ideally, I'd want to make this type "Todo", but since we don't have anything to handle parsing in this way, it's probably going to have to be "JSON", "Object" or "any".